kubectl/kubectl.go

```diff
@@ -778,6 +778,16 @@ func RunHelmBinaryWithCustomErr(args ...string) error {
 }
 
 // runBinary executes a binary cmd and returns the stdOutput and stdError combined
+// RunHelmBinaryWithOutput executes a desired binary and returns the output
+func RunHelmBinaryWithOutput(args ...string) (string, error) {
+	out, err := runBinary(helmCmd, args...)
+	if err != nil {
+		return string(out), &CustomError{strings.Join(append([]string{helmCmd}, args...), " "), string(out), err}
+	}
+	return string(out), nil
+}
+
+// runBinary executes a binary cmd and returns the stdOutput
```

Comment:
This comment should be removed and replaced by the one from L780, it seems that you mixed with the older version of this file ;-)